cscs-checks/prgenv/cuda/cuda_samples.py

```diff
@@ -8,14 +8,19 @@
 import reframe.utility.osext as osext
 
 
-class CudaSamples(rfm.RegressionTest):
+@rfm.simple_test
+class CudaSamplesTest(rfm.RegressionTest):
+    parameter('test_name', ['deviceQuery', 'concurrentKernels', 'simpleCUBLAS',
```

Comment:
`test_name` can be confused with the ReFrame test name. We should choose a better one.

Reply:
Also I will not merge this, because the discussion of the syntax has reopened in #1699 😬

Reply:
Do you think I should revert it to `@rfm.parameterized_test` syntax?

Reply:
No, we should just wait for #1699 to be merged, unless the tests are broken also for the systems that they are valid on.

Reply:
Yes, it gets a syntax error on tsa.

Reply:
Yes, revert it to `parameterized_test` syntax then.